examples/magnum/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-magnum)
+
+# The part below is adapted from
+# https://github.com/mosra/magnum-examples/blob/master/src/primitives/CMakeLists.txt
+
+# CMake policies. 0025 needs to be before project(), so putting all there.
+# Use AppleClang instead of Clang on Apple
+if(POLICY CMP0025)
+    cmake_policy(SET CMP0025 NEW)
+endif()
+# Don't treat imported targets with :: as files
+if(POLICY CMP0028)
+    cmake_policy(SET CMP0028 NEW)
+endif()
+# Enable MACOSX_RPATH by default
+if(POLICY CMP0042)
+    cmake_policy(SET CMP0042 NEW)
+endif()
+# Quoted variables should not be dereferenced
+if(POLICY CMP0054)
+    cmake_policy(SET CMP0054 NEW)
+endif()
+
+project(MagnumPrimitivesExample)
+
+# Add module path in case this is project root
+if(PROJECT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
+    set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${PROJECT_SOURCE_DIR}/../../modules/")
+endif()
```

Comment:
This is referencing a path that was only present in the magnum repositories, not sure if it makes any sense here.

Reply:
@mosra: Done!
Updated tests links:
https://ci.appveyor.com/project/pthom/hunter/builds/22140210
https://travis-ci.org/pthom/hunter/builds/489086544